### DISCUSSION TITLE
Assistance Needed to Obtain Required Files for ComfyUI

### DISCUSSION THREAD

**Discussion by AtlanticBiz** [comfyanonymous/ComfyUI#5438] [Q&A]
Hello ComfyUI team,

I am trying to obtain specific files (`clip_g.safetensors`, `clip_l.safetensors`, `t5xxl_fp16.safetensors`) necessary for my setup. I have been assigned the following app ID: `c53dd0ae-1fd8-471b-921d-04ed84f0d880`. Could you please advise on how to access or download these files? Thank you!

**Comment by LukeG89** [comment]
You can download those files from here: https://huggingface.co/Comfy-Org/stable-diffusion-3.5-fp8/tree/main/text_encoders
